shared/util/flow.tsx

```diff
@@ -1 +1,2 @@
-export const ifFlowComplainsAboutThisFunctionYouHaventHandledAllCasesInASwitch = (e: never) => {}
+// @ts-ignore codemod-issue
+export const ifFlowComplainsAboutThisFunctionYouHaventHandledAllCasesInASwitch = (e: never): never => {}
```

Comment:
i think we should toss this file

Reply:
We still need something like this for a similar reason. I was going to rename it in another pr.